### PR TITLE
Added relationships

### PR DESCRIPTION
**Pull request by theBwoyProgrammer**
## OOP School Library: Setup and Associations
- Create a class `Classroom` with `@label` instance variable initialized in the constructor.
- Created `Setter` and `getter` for `@label`
- Create a class Book `@title` and `@author` instance variables initialized in the constructor.
- Created `Setters` and `getters` for instance variables
- Create a class `Rental` with `@date` instance variable initialized in the constructor
- Created `Setter` and `getter` for `@date`
- Added relationships as asked

### CODE REVIEW

File: classroom.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def student(param)
```
Comment (Ismailco) [inline classroom.rb]: - [ ] [OPTIONAL] Kindly make sure the name of the method is descriptive, you can rename this method to `add_student` instead

File: student.rb [hunk 2]
```diff
@@ -9,6 +9,11 @@ def initialize(classroom = 'unknown')
   def play_hooky
     '¯\(ツ)/¯'
   end
+
```
Comment (Ismailco) [inline student.rb:14]: - [ ] [OPTIONAL] The classrom should be readable, kindly consider adding `attr_reader :classroom` to tour file

File: book.rb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+class Book
+  attr_accessor :author, :title, :rentals
+
+  def initialize(author, title)
+    @author = author
+    @title = title
+    @rentals = []
+  end
```
Comment (Ismailco) [inline book.rb:8]: - [ ] A book has many rentals, kindly consider adding this association by creating a method add_rental
e.g
```ruby
def add_rental(person, date)
    Rental.new(date, self, person)
  end
```

File: person.rb [hunk 4]
```diff
@@ -29,14 +29,15 @@ def correct_name
...
+  attr_accessor :name, :age, :rentals
 
   def initialize(age, name, parent_permission: true)
     super()
     @id = Random.rand(1..1000)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
```
Comment (Ismailco) [inline person.rb:42]: - [ ] A persone has many rentales kindly consider adding this relation to the person class
e.g
```ruby
 def add_rental(book, date)
    Rental.new(date, book, self)
 end
```

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @techwolf404, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Gitfow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by phelian23 — COMMENTED** [review]
Hi @techwolf404,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that is failing in linters. Kindly fix it and resubmit

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **Does not count** into the code reviews limit._

**Review by phelian23 — APPROVED** [review]
Hi @techwolf404, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
All linters passing ✔️
All requirements completed ✔️
Professional readme ✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**